src/sdk/clients/createNexusClient.ts

```diff
@@ -212,6 +211,8 @@ export async function createNexusClient(
     key,
     name,
     account: nexusAccount,
+    paymaster,
```

Comment:
Remove these. I think you're removing some changes that I've made by accident somehow.

Reply:
Remove what exactly ? The "paymaster" ?

Reply:
I meant the line below this, sorry